app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoSpeedController.kt

```diff
@@ -0,0 +1,41 @@
+package org.jellyfin.androidtv.ui.playback
+
+class VideoSpeedController(playbackController: PlaybackController) {
+	companion object {
+		enum class SpeedSteps(val speed: Double) {
```

Comment:
Move the speedsteps enum out of the companion object to make it easier accessible (VideoSpeedController.SpeedSteps instead of VideoSpeedController.Companion.SpeedSteps)